### PR TITLE
Adds the i belong static page 

### PR DESCRIPTION
**Pull request by nyashamaphosa**
## Status

* Current Status: Ready for review
* Related to https://github.com/NCCE/teachcomputing.org-issues/issues/2424
* Review app link https://teachcomputing-pr-1750.herokuapp.com/i-belong

## Review progress
- [x] Browser tested
- [x] Tech review
- [ ] Front end review

## What's changed?

Adds the I belong static page

### CODE REVIEW

File: app/views/pages/i-belong.html.erb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                    </h2>
+                    <ul class="govuk-body">
+                        <li>Develop an understanding of the factors affecting girls’ participation in computer science.</li>
+                        <li>Work with colleagues to produce an action plan with strategies to make computer science in your school more inclusive.</li>
+                    </ul>
+                    </div>
+                </div>
+                <div class="lp-support__card white-bg lp-support__card--bordered">
+                    <div>
+                    <h2 class="govuk-heading-m">
```
Comment (dcorking) [inline app/views/pages/i-belong.html.erb:46]: check your indentation settings in your editor, as this should be indented from line 46 to the end of the div

File: app/views/pages/i-belong.html.erb [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                                    link_url:'/login',
+                                    }
+                                },
+                                {
+                                    image_url: 'pages/i-belong/text_icon.png',
+                                    title: 'Gender Balance in computing research',
+                                    text: 'Read our reports from the Gender Balance in computing research project.',
+                                    link: {
+                                    link_title: 'Explore the research',
+                                    link_url: '/login',
```
Comment (dcorking) [inline app/views/pages/i-belong.html.erb:108]: This doesn't look right. See https://github.com/NCCE/teachcomputing.org-issues/issues/2424 :
 
> Resources for support - this should be accessible upon enrolment to the program except GBIC research. The GBIC link in the About Us Navigation bar should be removed and linked here. The only change will be an update to the text in the ‘Programme at a glance’ to
>  ‘This research project has now concluded. For information about our new programme to support girls into computer science, visit our [I Belong](https://teachcomputing.org/I-belong) page.

**Review by dcorking — CHANGES_REQUESTED** [review]
Hi, this page looks great in mobile and desktop.
Please review the comments below and also:
1. In mobile view, please put some whitespace between the students.jpg image and the orange bar at the start of the `lp-support__card`
Screenshot 2023-08-01 at 13 24 27

2. Pls aim to horizontally align the top of the text in the left hand card with the text in the other two cards to match the design (it looks like this is because the title in this card only takes up one line.) 
Screenshot 2023-08-01 at 13 38 50

File: spec/views/pages/i-belong.html_spec.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    expect(rendered).to have_css('.non-bordered-card', count: 3)
+  end
+
+#   it '2 grey boxes' do
+#     expect(rendered).to have_css('.padded-box', count: 2)
+#   end
+
+#   it 'has a learn more button' do
+#     expect(rendered).to have_link('Discover our research articles', href: 'https://blog.teachcomputing.org/tag/gbic-article/')
+#   end
```
[the remark this answers is not in the document]
Comment (nyashamaphosa) [reply spec/views/pages/i-belong.html_spec.rb:38]: resolved thank you

File: app/views/pages/i-belong.html.erb (hunk 1 above)
Comment (nyashamaphosa) [reply app/views/pages/i-belong.html.erb:46]: resolved too thank you

File: app/views/pages/i-belong.html.erb (hunk 2 above)
Comment (nyashamaphosa) [reply app/views/pages/i-belong.html.erb]: resolved too thank you

**Comment by nyashamaphosa** [comment]
> Hi, this page looks great in mobile and desktop. Please review the comments below and also:
> 
> 1. In mobile view, please put some whitespace between the students.jpg image and the orange bar at the start of the `lp-support__card`
> 
> Screenshot 2023-08-01 at 13 24 27
> 2. Pls aim to horizontally align the top of the text in the left hand card with the text in the other two cards to match the design (it looks like this is because the title in this card only takes up one line.)
> 
> Screenshot 2023-08-01 at 13 38 50

Resolved this bit too thank you

**Comment by nyashamaphosa** [comment]
> Hi, this page looks great in mobile and desktop. Please review the comments below and also:
> 
> 1. In mobile view, please put some whitespace between the students.jpg image and the orange bar at the start of the `lp-support__card`
> 
> Screenshot 2023-08-01 at 13 24 27
> 2. Pls aim to horizontally align the top of the text in the left hand card with the text in the other two cards to match the design (it looks like this is because the title in this card only takes up one line.)
> 
> Screenshot 2023-08-01 at 13 38 50

Resolved thank you. We might have to create another issue to resolve the second point as it affect a component shared by other pages.

File: app/views/pages/i-belong.html.erb (hunk 2 above)
Comment (dcorking) [reply app/views/pages/i-belong.html.erb]: I'll push a commit to feature flag this last sentence so it turns on at the same time as your I Belong page.

File: app/components/non_bordered_cards_component/non_bordered_cards_component.scss [hunk 4]
```diff
@@ -42,12 +42,14 @@
 .non-bordered-card__image {
   width: 100%;
   padding-bottom: 0;
+
+  &--bottom-border {
+    border-bottom: 8px solid #F47B31;
+  }
```
Comment (dcorking) [inline app/components/non_bordered_cards_component/non_bordered_cards_component.scss:48]:
```suggestion
```
Pls remove the orange `--bottom-border` CSS class as I don't think you used it.

**Review by dcorking — CHANGES_REQUESTED** [review]
Pls make these small changes and put the ticket on hold until Sam gets back to tomorrow. If he wants us to customize the nonbordered card component, I think it would be a tidy enough solution to add an optional parameter for the margin/padding that matches his design.

File: app/components/non_bordered_cards_component/non_bordered_cards_component.scss (hunk 4 above)
Comment (nyashamaphosa) [reply app/components/non_bordered_cards_component/non_bordered_cards_component.scss:48]: resolved thank you

**Review by dcorking — APPROVED** [review]
LGTM 🚂 👍

**Pull request merged**